sdk/documenttranslation/Azure.AI.DocumentTranslation/src/Azure.AI.DocumentTranslation.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <Description>Translator is a cloud-based machine translation service and is part of the Azure Cognitive Services family
+    of cognitive APIs used to build intelligent apps. Translator is easy to integrate in your applications, websites, tools,
+    and solutions. It allows you to add multi-language user experiences in more than 70 languages, and can be used on any
+    hardware platform with any operating system for text translation.</Description>
+    <AssemblyTitle>Microsoft Azure.AI.DocumentTranslation client library</AssemblyTitle>
+    <Version>1.0-preview.1</Version>
+    <PackageTags>Microsoft Azure Document Translation</PackageTags>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <!-- AZC0012 - Single word class names are too generic for Entity.cs class  -->
+    <NoWarn>$(NoWarn);3021;AZC0012</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" />
+    <PackageReference Include="System.Text.Json" />
+  </ItemGroup>
+
+  <!-- Import the Azure.Core project -->
+  <Import Project="$(MSBuildThisFileDirectory)..\..\..\core\Azure.Core\src\Azure.Core.props" />
+  <ItemGroup>
+    <Compile Include="$(AzureCoreSharedSources)Argument.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
```

Comment:
also, looks like `LinkBase ` will be better to use here. See discussion https://github.com/Azure/azure-sdk-for-net/pull/19292#discussion_r589787003